charts/argo-cd/Chart.yaml

```diff
@@ -3,7 +3,7 @@ appVersion: v2.7.6
 kubeVersion: ">=1.23.0-0"
 description: A Helm chart for Argo CD, a declarative, GitOps continuous delivery tool for Kubernetes.
 name: argo-cd
-version: 5.36.6
+version: 5.36.8
```

Comment:
merge dependency? 🤔 
https://github.com/argoproj/argo-helm/pull/2127

Reply:
Yeah - I didn't know how to handle that.  Can/should we bundle the two PR's in a single release?

Reply:
IMO
- It's better to separate release to reduce impact for users.
- We cannot promise which PR is merged ahead (this PR or your other PR or someone's PR), so I think this should be `5.36.7` for now

Reply:
I did 5.36.8 for this one, because #2127 has already been approved and I figured it would be merged first.

Reply:
Though we need one more approve to merge, I'll leave it to you.